regression/pages/studio/utils.py

```diff
@@ -0,0 +1,18 @@
+"""
+Utility functions for studio page objects.
+"""
+from opaque_keys.edx.locator import CourseLocator
+
+
+def get_course_key(course_info, module_store='split'):
+    """
+    Returns the course key based upon course info passed.
+    Course key depends upon the module store passed. The default
```

Comment:
Yes, it would be different. For example, when we use 'split' eventual url in the page would be studio.stage.edx.org/course_info/course-v1:ArbiRaees+AR-1000+fall but  studio.stage.edx.org/course_info/ArbiRaees/AR-1000/fall when using 'draft'. studio.stage.edx.org/course_info/ArbiRaees/AR-1000/fall (The url got using 'draft') would cause 403 forbidden error.